packages/shared/reactiveOmit/index.ts

```diff
@@ -0,0 +1,16 @@
+import type { UnwrapRef } from 'vue-demi'
+import { reactivePick } from '../reactivePick'
+
+/**
+ * Reactively omit fields from a reactive object
+ *
+ * @see https://vueuse.js.org/reactiveOmit
+ */
+export function reactiveOmit<T extends object, K extends keyof T>(
+  obj: T,
+  ...keys: K[]
+): { [S in Exclude<keyof T, K>]: UnwrapRef<T[S]>; } {
+  const pickKeys = Object.keys(obj).filter(k => !keys.includes(k as any)) as Array<Exclude<keyof T, K>>
+
+  return reactivePick(obj, ...pickKeys)
```

Comment:
I guess this is not ideal as it won't cover new keys added to that object

Reply:
You're right, I didn't take into account this situation, let me think about it.